src/test/java/org/broadinstitute/consent/http/service/DataAccessRequestServiceTest.java

```diff
@@ -782,11 +750,199 @@ void testValidateNoKeyPersonnelDuplicatesSO() {
         () -> service.validateNoKeyPersonnelDuplicates(data));
   }
 
+  @Test
+  void testValidatePersonnelInSameInstitutionThrowsException() {
+    String badEmailAddress = "j@example.com";
+    User user = new User(1, "email@test.org", "Display Name", new Date());
+    Institution usersInstitution = new Institution();
+    usersInstitution.setId(1);
+    user.setInstitution(usersInstitution);
+    DataAccessRequestData data = new DataAccessRequestData();
+    data.setPiEmail(badEmailAddress);
+    data.setSigningOfficialEmail(badEmailAddress);
+    data.setItDirectorEmail(badEmailAddress);
+    Collaborator collaborator = new Collaborator();
+    collaborator.setEmail(badEmailAddress);
+    data.setInternalCollaborators(List.of(collaborator));
+    Collaborator labStaffMember = new Collaborator();
+    labStaffMember.setEmail(badEmailAddress);
+    data.setLabCollaborators(List.of(labStaffMember));
+
+    Institution badInstitution = new Institution();
+    badInstitution.setId(2);
+    when(institutionService.findInstitutionForEmail(badEmailAddress)).thenReturn(badInstitution);
+    initService();
+    assertThrows(IllegalArgumentException.class, () -> service.validatePersonnelInSameInstitution(user, data));
```

Comment:
The code is very careful to format an error message with all the users and the field names but that behavior is not asserted here. `assertThrows()` returns the thrown exception, so this should also assert that the error message is as expected.